### DISCUSSION TITLE
雹 Hail v1.8.0

### DISCUSSION THREAD

**Discussion by aistra0528** [aistra0528/Hail#198] [Announcements]
- 新增九键搜索设置，感谢 @aeghn
- 新增首页字体大小设置，感谢 @hanks-zyh
- 优化应用界面，感谢 @Jesse205
- 优化自动冻结服务，感谢 @Henry-ZHR
- 修复 Android 14 快捷设置操作
- 更新印度尼西亚语翻译，感谢 @cikom0、Weblate@dedy prasetyo
- 更新日语翻译，感谢 Weblate@flippingpuppy
- 更新奥里亚语翻译，感谢 @SubhamJena
- 更新葡萄牙语翻译，感谢 @SantosSi
- 更新葡萄牙语（巴西）翻译，感谢 @Suburbanno
- 更新俄语翻译，感谢 @kilimov25
- 更新西班牙语翻译，感谢 @gallegonovato
- 更新土耳其语翻译，感谢 @forkposix、@oersen
- 更新乌克兰语翻译，感谢 @kilimov25、Weblate@Сергій
---
- T9 search support, thanks to @aeghn
- Customized Home font size, thanks to @hanks-zyh
- UI improvements, thanks to @Jesse205
- Enhanced auto freeze service, thanks to @Henry-ZHR
- Fix tile action on Android 14
- Update Indonesian translation, thanks to @cikom0, Weblate@dedy prasetyo
- Update Japanese translation, thanks to Weblate@flippingpuppy
- Update Odia translation, thanks to @SubhamJena
- Update Portuguese translation, thanks to @SantosSi
- Update Portuguese (Brazil) translation, thanks to @Suburbanno
- Update Russian translation, thanks to @kilimov25
- Update Spanish translation, thanks to @gallegonovato
- Update Turkish translation, thanks to @forkposix, @oersen
- Update Ukrainian translation, thanks to @kilimov25, Weblate@Сергій

<hr /><em>This discussion was created from the release <a href='https://github.com/aistra0528/Hail/releases/tag/v1.8.0'>雹 Hail v1.8.0</a>.</em>

**Comment by 9pyuco7t** [comment]
Unable to update 

> App not installed as package conflicts with an existing package.

Poco f5, Android 13, miui 14

**Reply by aistra0528** [reply]
Just wait for another few days till F-Droid updates. The F-Droid version uses a different app signature than GitHub version.
